spring-cloud-task-core/src/main/java/org/springframework/cloud/task/listener/DefaultTaskExecutionKeyValuesProvider.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.listener;
+
+import io.micrometer.common.KeyValues;
+
+import org.springframework.cloud.task.repository.TaskExecution;
+
+/**
+ * /**
+ * Default {@link TaskExecutionKeyValuesProvider} implementation.
+ *
+ * @author Glenn Renfro
+ * @since 3.0.0
+ */
+public class DefaultTaskExecutionKeyValuesProvider implements TaskExecutionKeyValuesProvider {
+
+	@Override
+	public KeyValues getLowCardinalityKeyValues(TaskExecutionObservationContext context) {
+		TaskExecution execution = context.getTaskExecution();
+		return KeyValues.of(TaskExecutionMeters.TaskTags.TASK_NAME_TAG.of(execution.getTaskName()),
```

Comment:
Weren't these moved to the place when the `Observation` is created? If so, we might not need this class anymore.